Core/src/Geometry/CompositePortalLink.cpp

```diff
@@ -248,7 +248,7 @@ std::unique_ptr<GridPortalLink> CompositePortalLink::makeGrid(
 
     auto grid = GridPortalLink::make(m_surface, m_direction, std::move(axis));
     for (const auto& [i, child] : enumerate(trivialLinks)) {
-      grid->atLocalBins({i + 1}) = &child->volume();
+      grid->grid().atLocalBins({i + 1}) = &child->volume();
```

Comment:
`grid->grid()` reads a bit weird now.

Reply:
`grid` here is the `gridPortalLink` right? If so, you could name it `gridPLink`.

Reply:
I just called it `gridPortalLink` now, that seems to work as well.